Makefile.frag

```diff
@@ -1,2 +1,8 @@
+format-code:
+	clang-format -i *.{c,h} src/jsonpath/*.{c,h}
```

Comment:
Perhaps leave out `jsonpath_arginfo.h` and `jsonpath_legacy_arginfo.h`, as they are auto-generated with a certain style? I guess that means `clang-format -i jsonpath.c php_jsonpath.h src/jsonpath/*.{c,h}`, or some CLI magic to exclude the arginfo files from the wildcard.

Reply:
Makes sense, done. 👍